shared/middleware/helpers/storage_apps.rb

```diff
@@ -14,7 +14,7 @@ def initialize(storage_id)
     @table = PEGASUS_DB[:storage_apps]
   end
 
-  def create(value, ip:, type: nil, published_at: nil, remix_parent_id: nil)
+  def create(value, ip:, type: nil, published_at: nil, remix_parent_id: nil, skip_content_moderation: false)
```

Comment:
Do we ever need to create a storage_apps row with `skip_content_moderation: true`?  If so, let's add tests for this case.  If not, maybe we don't want it to be a parameter here.